### PR TITLE
Dynamically generate styles

### PR DESCRIPTION
**Pull request by davidebbo**
This is to address adding more colors, per #7. I started with the existing set of colors, and picked a few more to get to 26.

Hopefully, this is more maintainable, since a color can be added simply by adding one element to an array.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -561,6 +561,36 @@ <h1>{{ { US: 'USA', Intl: 'International' }[chosen_domain] }}
...
+              "#86797d", "#b2c326", "#6188e2", "#a748ca", "#a7f432", "#a83731", "#af6e4d", "#ff007c", "#fcd667", "#bfafb2",
+              "#c46210", "#4a646c", "#a6e7ff", "#ffd0b9", "#45a27d", "#3aa8c1"]
+
+//////////////////////////////////////////////////////////////////////
+// Dynamically create all styles, to make sure the ones from the chart
+// and from the legend match
+//////////////////////////////////////////////////////////////////////
+function createCssStyles() {
+  for(var i=0; i < colors.length; i++) {
+    // Using style injection technique from https://stackoverflow.com/a/1720483/111888
```
Comment (davidbau) [inline index.html]: Fancy!

**Comment by davidebbo** [comment]
I think we can easily cycle with a mod on this line: https://github.com/davidbau/covid-19-chart/blob/c47b06e351d88dcaa955540c4b490cd80dfaa6d5/lib/chartist-plugin-legend.js#L116

But the hard part is that this code doesn't know how many colors there are, and passing that through via plugin settings feels overkill. Maybe we can just hard code `i % 26` and call it a day?

**Comment by davidebbo** [comment]
I ended up just passing the max count after all...

**Comment by davidbau** [comment]
Thanks @davidebbo!  Here are all 50 states on one plot.  https://covid19chart.org/#/?top=50

image

**Comment by davidbau** [comment]
Actually, https://covid19chart.org/#/?top=53 - all 50 states plus DC, Puerto Rico, and Guam.

We are all one country in the end, sharing the same diseases.

**Pull request merged**